src/views/admin-results-listing.phtml

```diff
@@ -46,22 +46,23 @@
                     <!--Sorting arrow are displayed using the 'asc','dsc' classes-->
                     <th scope="col" class="sorting-header asc dsc">ID</th>
                     <th scope="col">User</th>
-                    <th scope="col">Quiz</th>
-                    <!--Sorting arrow are displayed using the 'asc','dsc' classes-->
-                    <th scope="col" class="sorting-header asc dsc">Score</th>
+                    <th scope="col">QuizName</th>
```

Comment:
here, you are in user land :D users don't enjoy camel case/title case/snake case, use "Quiz name" better